### ISSUE TITLE
OCP popup fails to display

### ISSUE THREAD

**Issue by puxlit** [new-xkit/XKit#1476]
### Environment

  - Browser: Firefox (tested 57, 59)
  - Extension: New XKit (tested 7.8.1)
  - XKit Extension: One-Click Postage (tested 4.3.6)
  - Extension: Textarea Cache (tested 3.3.0)

### Steps to reproduce

 1. Have Textarea Cache ([AMO](https://addons.mozilla.org/en-US/firefox/addon/textarea-cache/); [GH](https://github.com/wildskyf/TextareaCache)) and New XKit (with OCP) installed.
 2. Load the dashboard, trigger the OCP popup, enter some text into the caption field, then refresh the page.
 3. Attempt to trigger the OCP popup.

### Expected results

OCP popup appears.

### Actual results

Nothing seems to happen.

### Further details

This conflict was first reported by `Primarybufferpanel#6870`.

After some printf() debugging, we narrowed down the failing line to `$("#x1cpostage_caption").blur();` in `XKit.extensions.one_click_postage.reset_box`.

Debug patch follows.

```diff
diff --git a/Extensions/one_click_postage.js b/Extensions/one_click_postage.js
index 3aac83e..e43b513 100644
--- a/Extensions/one_click_postage.js
+++ b/Extensions/one_click_postage.js
@@ -1076,7 +1076,13 @@ XKit.extensions.one_click_postage = new Object({
                $("#x1cpostage_caption").val("");
                $("#x1cpostage_tags").val("");
                $("#x1cpostage_tags").blur();
+               $('#x1cpostage_caption').parents().addBack().one('blur', function (event) {
+                       console.debug('Blur bubbled up to:', event.currentTarget);
+               });
+               console.log('One-shot blur event handlers for #x1cpostage_caption prepared');
+               console.debug('About to blur #x1cpostage_caption');
                $("#x1cpostage_caption").blur();
+               console.debug('NB: we never reach this line');
                XKit.extensions.one_click_postage.auto_tagger_done = false;
        },

```

Combined web and browser console outputs follow.

```
[xkit/one_click_postage.js:1082:3] One-shot blur event handlers for #x1cpostage_caption prepared
[xkit/one_click_postage.js:1083:3] About to blur #x1cpostage_caption
[xkit/one_click_postage.js:1080:4] Blur bubbled up to: <textarea id="x1cpostage_caption" class="ta-textContent" placeholder="caption" data-id="2">
[jquery.js:3:5595                ] Error: Permission denied to access property "apply"
```

Notably, whilst Textarea Cache may incite this misbehaviour, OCP is failing within the version of jQuery (2.1.4) that _XKit_ bundles.

`jquery.js:3:5595` maps to the `handle.apply` in [dist/jquery.js:4355](https://github.com/jquery/jquery/blob/7751e69b615c6eca6f783a81e292a55725af6b85/dist/jquery.js#L4355) (or [src/event.js:310](https://github.com/jquery/jquery/blob/7751e69b615c6eca6f783a81e292a55725af6b85/src/event.js#L310)).

What's inaccessible is the native `onblur` property of `#x1cpostage_caption`, which is normally `null` but, thanks to Textarea Cache (via [content-script.js:57](https://github.com/wildskyf/TextareaCache/blob/2c810d508543723e81cdd181cba0d9762899fa2f/content-script.js#L57)), has been assigned.

Chrome better isolates on-event handlers between pages and content scripts, so this appears to be a Firefox issue, and was reported to BMO as [#1330113](https://bugzilla.mozilla.org/show_bug.cgi?id=1330113).

**Comment by puxlit** [comment]
`jquery.js:3:5595` maps to the `handle.apply` in [jquery-2.1.4.js:4355](https://github.com/jquery/jquery/blob/7751e69b615c6eca6f783a81e292a55725af6b85/dist/jquery.js#L4355).

**Comment by puxlit** [comment]
### Things I Gotta Do

  - [ ] Determine whether this behaviour is intentional of FF's Xray vision, or constitutes a bug.
  - [ ] Compare behaviour against Chrome.

**Comment by puxlit** [comment]
Fixed in Textarea Cache 3.5.0 (by wildskyf/TextareaCache@b0610a1200e8a6fe45a5e4af055fa010038fde93).
